<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage</class>
 <widget class="QMainWindow" name="Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后台管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QTabWidget" name="MainSelect">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>901</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:299px; height:50px; background-color:rgb(255,255,255)}
QTabBar::tab::pane{border:1px solid rgb(115, 115, 115);
border-bottom:0px; border-left:0px}
QTabWidget:pane{border: none;}
QTabBar::tab:selected{background-color: rgb(230, 230, 230);}
QTabBar::tab:!selected{background-color: rgb(255, 255, 255);}


</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Things_m">
     <attribute name="title">
      <string>菜品管理</string>
     </attribute>
     <widget class="QTabWidget" name="DishsSelect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{width:149px; height:50px; background-color:rgb(255,255,255); }
QTabBar::tab::pane{border:1px solid rgb(222,222,222);
border-bottom:0px; border-left:0px}
QTabWidget:pane{border: none;}
QTabBar::tab:selected{background-color: rgb(204, 229, 255);}
QTabBar::tab:!selected{background-color: rgb(255, 255, 255);}



</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>米饭</string>
       </attribute>
       <widget class="QListView" name="Show_Rice">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>面食</string>
       </attribute>
       <widget class="QListView" name="Show_Noodle">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>汤</string>
       </attribute>
       <widget class="QListView" name="Show_Soup">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>911</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>小吃</string>
       </attribute>
       <widget class="QListView" name="Show_Snack">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>酒水</string>
       </attribute>
       <widget class="QListView" name="Show_Drink">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>小炒</string>
       </attribute>
       <widget class="QListView" name="Show_Fire">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Add_dish">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.0168182, x2:1, y2:0, stop:0 rgba(29, 187, 224, 255), stop:1 rgba(154, 255, 255, 255));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(54, 227, 236, 255), stop:1 rgba(199, 255, 255, 255));
border-width:2px;
border-style:solid;
border-radius:10px</string>
      </property>
      <property name="text">
       <string>添加菜品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete_dish">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>500</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.0168182, x2:1, y2:0, stop:0 rgba(29, 187, 224, 255), stop:1 rgba(154, 255, 255, 255));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(54, 227, 236, 255), stop:1 rgba(199, 255, 255, 255));
border-width:2px;
border-style:solid;
border-radius:10px</string>
      </property>
      <property name="text">
       <string>删除菜品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Change_dish">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>500</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.0168182, x2:1, y2:0, stop:0 rgba(29, 187, 224, 255), stop:1 rgba(154, 255, 255, 255));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(54, 227, 236, 255), stop:1 rgba(199, 255, 255, 255));
border-width:2px;
border-style:solid;
border-radius:10px</string>
      </property>
      <property name="text">
       <string>修改菜品</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Orders_m">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QListWidget" name="Show_Orders">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px;</string>
      </property>
     </widget>
     <widget class="QListView" name="Order_Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Change_Order">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>480</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.0168182, x2:1, y2:0, stop:0 rgba(29, 187, 224, 255), stop:1 rgba(154, 255, 255, 255));
border:2px solid rgba(154, 255, 255, 255);
border-radius:15px;</string>
      </property>
      <property name="text">
       <string>修改订单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete_Order">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>480</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(242, 85, 14, 255), stop:1 rgba(255, 206, 37, 255));
border:2px solid rgba(255, 206, 37, 255);
border-radius:15px;</string>
      </property>
      <property name="text">
       <string>删除订单</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Total_m">
     <attribute name="title">
      <string>销量统计</string>
     </attribute>
     <widget class="QListView" name="statius_dish">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>441</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-top:0px;</string>
      </property>
     </widget>
     <widget class="QListView" name="statius_order">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>431</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-top:0px;
border-right:0px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-bottom:0px;</string>
      </property>
      <property name="text">
       <string>菜品销量统计</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-bottom:0px;
border-right:0px;
</string>
      </property>
      <property name="text">
       <string>顾客消费统计</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>总销售额</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>最受欢迎菜品</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="total_income">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-radius: 5px;
color: rgb(245, 9, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="most_popular">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(222, 222, 222);
border-radius: 5px;
color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
